src/Microsoft.DocAsCode.MarkdownLite/Regexes.cs

```diff
@@ -110,7 +110,8 @@ public static class Gfm
                 public static readonly Regex StrongEm = new Regex(@"^(\**?)\*{3}(?!\*)(?=\S)([\s\S]*?\S)\*([\s\S]*?(?<=\S))?\*{2}", RegexOptionCompiled);
                 public static readonly Regex Strong = new Regex(@"^__([\s\S]+?)__(?!_)|^\*{2}(?!\*)(?=\S)([\s\S]*?\S)?\*{2}", RegexOptionCompiled);
                 public static readonly Regex Em = new Regex(@"^\b_((?:__|[\s\S])+?)_\b|^\*((?:\*\*|\S[\s\S]*?))(?<!\*)\*", RegexOptionCompiled);
-                public static readonly Regex Text = new Regex(@"^[\s\S]+?(?=[\\<!\[_*`~]|https?:\/\/| {2,}\n|$)", RegexOptionCompiled);
+                public static readonly Regex Emoji = new Regex(@"^\:([a-z_]+)\:", RegexOptionCompiled);
```

Comment:
emoji can also contain number?

---
In reply to: [71101226](https://github.com/dotnet/docfx/pull/502#discussion_r71101226) [](ancestors = 71101226)